storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -29,6 +29,44 @@ In future {product-title} versions, volumes provisioned using existing in-tree p
 After full migration, in-tree plug-ins will eventually be removed in future versions of {product-title}.
 ====
 
+=== Storage policy
```

Comment:
Sections could use anchor links if they're left as-is.

Reply:
Decided to create modules.